src/driver.rs

```diff
@@ -1,6 +1,5 @@
 #![cfg_attr(feature = "deny-warnings", deny(warnings))]
 #![feature(rustc_private)]
-#![feature(str_strip)]
 
 // FIXME: switch to something more ergonomic here, once available.
```

Comment:
I can't see the original line in the master branch, maybe you need to rebase?

Reply:
rebased my branch, this diff doesn't show up anymore